### PR TITLE
added inferno-context-api-store to libraries list.

### PR DESCRIPTION
**Pull request by aprilmintacpineda**
I created and will maintain this library. I also use it in our project.

### CODE REVIEW

File: src/docs/guides/libraries.md [hunk 1]
```diff
@@ -153,4 +154,183 @@ const PopperExample = () => (
     </Popper>
   </Manager>
 )
+```
+
+<a name="inferno-context-api-store"></a>
+## inferno-context-api-store
+
+Seemless, light weight, state management library that comes with asynchronous support out of the box. Inspired by Redux and Vuex. Built on top of [inferno-create-context](https://github.com/kurdin/create-inferno-context).
```
Comment (Havunen) [inline src/docs/guides/libraries.md]: typo: lightweight

File: src/docs/guides/libraries.md [hunk 2]
```diff
@@ -153,4 +154,183 @@ const PopperExample = () => (
...
+
+The file size is roughly 5.1kb transpiled, not minified.
+
+Read the docs at https://github.com/aprilmintacpineda/inferno-context-api-store.
+
+### Usage
+
+Usage is the same as with redux. Except I used React's new Context API in version 16.3.0. I also simplified some stuff. If you've used Redux and Vuex in the pass, everything here will be familiar to you.
```
Comment (Havunen) [inline src/docs/guides/libraries.md]: Lets remove "I also simplified some stuff" or re-phrase it

Comment (Havunen) [inline src/docs/guides/libraries.md]: and Vuex in the pass => typo: past

**Comment by Havunen** [comment]
Hi @aprilmintacpineda Thanks for the PR! I did some proof reading, can you fix those and lets merge it! LGTM otherwise :+1:

**Comment by aprilmintacpineda** [comment]
@Havunen thank you so much for that!

**Comment by Havunen** [comment]
LGTM thanks! :)

**Pull request merged**